src/encoded/static/components/GermlineTableNoState.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable linebreak-style */
+// eslint-disable-next-line linebreak-style
+import React from 'react';
+import { SortTablePanel, SortTable } from './sorttable';
+
+class GermlineTableNoState extends React.Component {
+    constructor(props) {
+        super(props);
+        // this.state = {
+        //     showData: true,
+        // };
+        //this.tableConfig = {};
+        this.data = this.props.data;

```

Comment:
Don't assign a property to a component variable. It's not a hard copy so you can't modify this.data anyway. Use this.props.data where this.data is used instead.

Reply:
I pulled a  wrong code and re-pushed on Saturday to correct it. Not sure how we can recall the wrong one back.  Sorry for the confusing. I will change the props. data and transformDataFunction . Any other suggestions are welcomed to address here.

Reply:
No need to recall. Its okay, this happens all the time.